src/Module/Contact.php

```diff
@@ -984,24 +984,26 @@ public static function getContactTemplateVars(array $rr)
 		$dir_icon = '';
 		$alt_text = '';
 
-		switch ($rr['rel']) {
-			case Model\Contact::FRIEND:
-				$dir_icon = 'images/lrarrow.gif';
-				$alt_text = L10n::t('Mutual Friendship');
-				break;
+		if ($rr['uid']) {
```

Comment:
Why no "empty(...)" here?

Reply:
You're right, I expect the `uid` field to be existing like the `rel` field is expected to exist but it may not be the case.